src/adapters/cloudflare.ts

```diff
@@ -108,4 +108,15 @@ class CloudflarePeer extends Peer<{
     this.ctx.cloudflare.server.send(toBufferLike(message));
     return 0;
   }
+
+  close(code?: number, reason?: string) {
+    this.ctx.cloudflare.client.close(code, reason);
```

Comment:
I'm not familiar with Cloudflare websockets and having two of them so this could be wrong